dataprofiler/profilers/utils.py

```diff
@@ -411,3 +411,21 @@ def find_diff_of_lists_and_sets(stat1, stat2):
         unique2 = [element for element in stat2 if element not in stat1]
         diff = [unique1, shared, unique2]
     return diff
+
+def timedelta_to_readable_output(diff):
```

Comment:
I think it might be better to just have a util function which is `find_diff_of_datetime` and to just have one call. Thoughts?

Reply:
Also, we will need a test specifically for this function or the new function described above.

Reply:
I feel like it's a bit redundant to find the differences between the dates within the function, because that would essentially be rewriting find_diff_of_numbers() inside the function.